tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js

```diff
@@ -194,15 +193,11 @@ describe("Feature: Repeating Sections with Hub and Spoke", () => {
       $(ProxyPage.noIMAnsweringForMyself()).click();
       $(ProxyPage.submit()).click();
 
-      expect($(DateOfBirthPage.questionText()).getText()).to.equal("What is your date of birth?");
-
```

Comment:
These tests are testing that the question text is proxy or not, so we should keep this expect and remove the rest of the lines of this test. I don't think we need to check two separate pages for the proxy text.